scalaparse/shared/src/test/scala/scalaparse/unit/FailureTests.scala

```diff
@@ -835,6 +835,16 @@ object FailureTests extends TestSuite{
         expected = """ "\"\"\"" | StringChars | Interp | NonTripleQuoteChar """,
         found = ""
       )
+      * - checkNeg(
+        """
+          |object System {
+          |  e match { case <xml:unparsed><</xml:unparsed> => }
```

Comment:
This test fails on 2.12 but not on 2.11 and 2.10.
```scala
expected  = CharsWhile(<function1>) | XmlPattern | Thingy |...
```
and
```scala
parsedExpected = CharsWhile(scalaparse.syntax.Identifiers$$$Lambda$5210/206499243@72995185) | XmlPattern | Thingy | ...
```
`toString` on lambda is different between 2.11 and 2.12